parser/proto.y

```diff
@@ -406,9 +405,6 @@ messageLiteralField : messageLiteralFieldName ':' value {
 			$$ = nil
 		}
 	}
-	| error {
-		$$ = nil
-	}
```

Comment:
Why remove this? If it's a parser ambiguity, I think `error ':' value` might be better and unambiguous. That at least lets the parser potentially recover from a syntax error in the field name error if it sees a colon.

Reply:
I removed the line cuz it produced `never used`  warning in goyacc. Changed this line to `error ':' value`.